jupyter_rfb/events.py

```diff
@@ -4,10 +4,9 @@
 Events are simple dict objects containing at least the key `event_type`.
 Additional keys provide more information regarding the event.
 
-Note that in jupyter_rfb, the *key_down*, *key_up*, and *wheel* events
-only apply when the widget has focus (having received a pointer down).
```

Comment:
Is this information no longer provided? Should it be?

Reply:
It has moved down to a separate subsection about focus.